### PR TITLE
[MRG+1] use imgmath extension when using sphinx >= 1.4

### PR DESCRIPTION
**Pull request by aabadie**
Fix fixes circle ci html strict documentation generation.

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -30,8 +31,12 @@
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.autosummary',
-              'sphinx.ext.pngmath', 'sphinx.ext.intersphinx',
+extensions = ['sphinx.ext.autodoc',
+              'sphinx.ext.autosummary',
+              'sphinx.ext.imgmath'  # only available for sphinx >= 1.4
+              if sphinx.version_info[:2] >= (1, 4)
+              else 'sphinx.ext.pngmath',
```
Comment (GaelVaroquaux) [inline doc/conf.py]: I would indent here (and maybe put the whole expression in parenthesis).

**Comment by GaelVaroquaux** [comment]
+1 for merge (maybe after addressing cosmetics)

File: doc/conf.py (hunk 1 above)
Comment (aabadie) [inline doc/conf.py]: Comment addressed using parenthesis.

**Comment by aabadie** [comment]
The [generated doc](https://circle-artifacts.com/gh/aabadie/nilearn/338/artifacts/0/home/ubuntu/nilearn/doc/_build/html/index.html) seems ok to me.

**Comment by KamalakerDadi** [comment]
How can I see the PR effect in circle CI ?

**Comment by KamalakerDadi** [comment]
Ok got it. LGTM too.

**Comment by lesteve** [comment]
Merging, thanks!

**Pull request merged**